schemas/+acquisition/Session.m

```diff
@@ -10,9 +10,13 @@
 stimulus_commit = ''        : varchar(64)                   # git hash for the version of the function
 stimulus_set                : tinyint                       # an integer that describes a particular set of stimuli in a trial
 session_performance         : float                         # percentage correct on this session
+num_trials=-1               : int(11)                       # Number of trials for the session
```

Comment:
what's the meaning of -1?

Reply:
In this case, -1 means that number of trials is not determined yet. 
(Sessions are being written right now without num_trials field)
I guess NULL is more appropriate for that.